client/components/domains/register-domain-step/index.jsx

```diff
@@ -46,6 +46,7 @@ const INITIAL_SUGGESTION_QUANTITY = 2;
 const analytics = analyticsMixin( 'registerDomain' ),
 	searchVendor = 'domainsbot';
 
+//region Query Reporting Optimization
```

Comment:
I'm guessing this is some sort of aid for an IDE? Probably should not commit this either way.